static/js/flow/discussionTypes.js

```diff
@@ -19,7 +19,7 @@ type ChannelType = "private" | "public";
 export type ChannelInputs = {
   name:               string,
   title:              string,
-  public_description: string,
+  description: string,
```

Comment:
Can you adjust the indentation here and below? I guess the reformatter isn't aware of flow type declarations

Reply:
Yeah, these need to be manually formatted actually cause the formatter actually removes the alignments.

Reply:
That's fine, let's just leave it then

Reply:
Fixed it anyway since it was small/quick